tf-modules/log-subscription-filter/ingest_log_subscriptions.tf

```diff
@@ -37,3 +37,11 @@ resource "aws_cloudwatch_log_subscription_filter" "sync_granule_task_log_subscri
   filter_pattern  = ""
   log_group_name  = var.sync_granule_task
 }
+
+resource "aws_cloudwatch_log_subscription_filter" "hyrax_metadata_updates_task_log_subscription_filter" {
```

Comment:
I haven't seen this in a lot of other PRs - what is this doing?

Reply:
Oh so this is to send the logs over to the cloud metrics account for ingest. For ingest, we want to configure sending the logs over so we can see the trip the granule took through the workflow tasks to get ingested. Since this task would get added to an ingest workflow I wanted to set that up.